src/test/regress/expected/intermediate_results.out

```diff
@@ -569,9 +569,46 @@ WARNING:  Query could not find the intermediate result file "squares_2", it was
 ---------------------------------------------------------------------
 (0 rows)
 
+-- test refreshing mat views
+SET client_min_messages TO ERROR;
+SELECT run_command_on_workers($$CREATE USER some_other_user;$$);
+      run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,"CREATE ROLE")
+ (localhost,57638,t,"CREATE ROLE")
+(2 rows)
+
+CREATE USER some_other_user;
+SELECT run_command_on_workers($$GRANT ALL ON DATABASE regression TO some_other_user;$$);
+  run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,GRANT)
+ (localhost,57638,t,GRANT)
+(2 rows)
+
+GRANT ALL ON DATABASE regression TO some_other_user;
+RESET client_min_messages;
+\c - some_other_user
+CREATE SCHEMA other_schema;
+SET search_path TO other_schema;
+CREATE TABLE dist_table (a int, b int);
```

Comment:
lets ingest few rows, we might in the future avoid files at all if no rows exists